scripts/ci.sh

```diff
@@ -35,7 +35,7 @@ setup_protoc() {
 	sudo mv protoc3/bin/* /usr/local/bin/
 	sudo mv protoc3/include/* /usr/local/include/
 	rm -f PROTOC_ZIP
-	go get -u github.com/golang/protobuf/protoc-gen-go \
+	GO111MODULE=off go get  -u github.com/golang/protobuf/protoc-gen-go \
```

Comment:
Please remove an extra space character here (`get  -u`). Also, please check why CI is failing.

Reply:
Yes, I will fix it.

Reply:
@1995parham this PR is complex, touches a lot of stuf and if we want it merged must be really well maintained. You must work every day with Mainflux developers until it is merged properly, otherwise it goes out of the sync and stays hanging there for months.

I propose that either we merge it on Monday, or we close it due specific and effort-demanding task, and let one of Mainflux core members to handle this while you switch to some other PRs that would be more isolated and easier to merge.

Reply:
@drasko Thanks for your comment. I think I can switch to another Issue and let it be merged by one of the core members. I will choose another issue based on Mainflux Gitter.